diff --git a/CMakeLists.txt b/CMakeLists.txt
index 20724676..e95822ad 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -358,11 +358,6 @@ if(NOT MSVC)
     message(STATUS "Default linker")
   endif()

-  include(CheckSSE4_2)
-  if( BUILTIN_POPCNT )
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -msse4.2")
-  endif()
-
   include(CheckCXXCompilerFlag)
   check_cxx_compiler_flag(-mcrc32 KAHYPAR_HAS_CRC32)
   if(KAHYPAR_HAS_CRC32 AND X86)
