diff --git a/cmake/modules/SetupInstallation.cmake b/cmake/modules/SetupInstallation.cmake
index 2b1af0b8..6ead7f5b 100644
--- a/cmake/modules/SetupInstallation.cmake
+++ b/cmake/modules/SetupInstallation.cmake
@@ -7,14 +7,9 @@ set_target_properties(mtkahypar PROPERTIES
 set_target_properties(mtkahypar PROPERTIES
     PUBLIC_HEADER "${MTKAHYPAR_INCLUDE_DIR}/mtkahypar.h;${MTKAHYPAR_INCLUDE_DIR}/mtkahypartypes.h")
 
-configure_file(cmake/mtkahypar.pc.in mtkahypar.pc @ONLY)
 configure_file(cmake/MtKaHyParConfig.cmake.in MtKaHyParConfig.cmake @ONLY)
 configure_file(cmake/MtKaHyParConfigVersion.cmake.in MtKaHyParConfigVersion.cmake @ONLY)
 
-install(FILES ${CMAKE_BINARY_DIR}/mtkahypar.pc
-  DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig
-  COMPONENT MtKaHyPar_Lib)
-
 install(FILES ${CMAKE_BINARY_DIR}/MtKaHyParConfig.cmake
               ${CMAKE_BINARY_DIR}/MtKaHyParConfigVersion.cmake
   DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/MtKaHyPar
